src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -53,13 +60,20 @@
 
 {% block form_row -%}
   {% spaceless %}
-    <div class="{{ block('form_row_class') }}{% if (not compound or force_error|default(false)) and not valid %} has-error{% endif %}">
-      {{ form_label(form) }}
-      <div class="{{ block('form_group_class') }}">
-        {{ form_widget(form) }}
-        {{ form_errors(form) }}
-      </div>
-    </div>
+        {# could be refactored better if we search id_state in block_prefixes and there's only one js-state class  #}
```

Comment:
Or maybe it should be managed in the FormType rather than in the generic template?